example_files/resources/hpxml-measures/HPXMLtoOpenStudio/resources/version.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 class Version
-  OS_HPXML_Version = '1.5.1' # Version of the OS-HPXML workflow
-  OS_Version = '3.5' # Required version of OpenStudio (can be 'X.X' or 'X.X.X')
+  OS_HPXML_Version = '1.6' # Version of the OS-HPXML workflow
+  OS_Version = '3.6' # Required version of OpenStudio (can be 'X.X' or 'X.X.X')
```

Comment:
You are manually editing these? If so, I would definitely not recommend that. The normal way would be to `git subtree pull` from the OS-HPXML repo directly...

Reply:
Ohhh, thank you! I did not know that was the correct workflow.